com.microsoft.mrtk.core/Utilities/Async/Internal/AsyncCoroutineRunner.cs

```diff
@@ -0,0 +1,175 @@
+// MIT License
+
+// Copyright(c) 2016 Modest Tree Media Inc
```

Comment:
Where does this all come from?

Reply:
Nvm, I see the readme. I really don't think this should all be brought into our Core package... do we really need all of these async utilities/extensions, especially if it's literally just for a single feature?

Reply:
Removed this package